pdfannots.py

```diff
@@ -552,6 +552,8 @@ def parse_args():
                    help=("sections to emit (default: %s)" % ', '.join(allsects)))
     g.add_argument("--no-group", dest="group", default=True, action="store_false",
                    help="emit annotations in order, don't group into sections")
+    g.add_argument("--print-filename", dest="print", default=False, action="store_true",
```

Comment:
args.print is a little ambiguous. Do you mind naming this variable "printfilename" or similar?

Reply:
(looks fine to me otherwise)

Reply:
Thanks for the suggestion. Of course, 'print' is a little ambiguous. So I've changed 'print' to 'printfilename'.